cpp/benchmarks/common/generate_input.cu

```diff
@@ -751,13 +752,16 @@ std::unique_ptr<cudf::column> create_random_column<cudf::list_view>(data_profile
 
   // Generate the list column bottom-up
   auto list_column = std::move(leaf_column);
-  for (int lvl = 0; lvl < dist_params.max_depth; ++lvl) {
+  for (int lvl = dist_params.max_depth; lvl > 0; --lvl) {
```

Comment:
Why are we moving backward?

Reply:
We've always generated these columns bottom-up, just the value of `lvl` did not match this. `lvl` is not used in the loop, outside of the new check.

Reply:
Ah that makes sense.